spec/requests/v1/unpublish_spec.rb

```diff
@@ -73,37 +73,42 @@
           FileUtils.rm_rf legacy_stacks_path
         end
 
-        it 'marks the purl as deleted and removes files' do
-          delete("/purls/#{druid}", headers:)
-
-          expect(response).to have_http_status(:success)
-
-          expect(purl_object.reload).to have_attributes(deleted_at: (a_value > 5.seconds.ago))
-          expect(Racecar).to have_received(:produce_sync)
-            .with(key: purl_object.druid, topic: 'testing_topic', value: nil)
-          expect(File).not_to exist(file_content_path)
-          expect(File).not_to exist(legacy_file_path)
-          expect(File).not_to exist(cocina_version_path)
-          expect(File).not_to exist(cocina_head_version_path)
-          expect(File).not_to exist(xml_version_path)
-          expect(File).not_to exist(xml_head_version_path)
+        context 'when no version is provided' do
+          it 'marks the purl as deleted and removes files' do
+            delete("/purls/#{druid}", headers:)
+
+            expect(response).to have_http_status(:success)
+
+            expect(purl_object.reload).to have_attributes(deleted_at: (a_value > 5.seconds.ago))
+            expect(Racecar).to have_received(:produce_sync)
+              .with(key: purl_object.druid, topic: 'testing_topic', value: nil)
+            expect(File).not_to exist(file_content_path)
+            expect(File).not_to exist(legacy_file_path)
+            expect(File).not_to exist(cocina_version_path)
+            expect(File).not_to exist(cocina_head_version_path)
+            expect(File).not_to exist(xml_version_path)
+            expect(File).not_to exist(xml_head_version_path)
+          end
         end
-      end
-
-      context 'when versioned files is enabled and version is provided' do
-        let(:service) { instance_double(VersionedFilesService, delete: true, versioned_files?: true) }
-
-        before do
-          allow(Settings.features).to receive(:versioned_files).and_return(true)
-          allow(VersionedFilesService).to receive_messages(new: service)
-        end
-
-        it 'invokes VersionedFilesService#delete with the version' do
-          delete("/purls/#{druid}?version=2", headers:)
 
-          expect(response).to have_http_status(:success)
-          expect(VersionedFilesService).to have_received(:new).with(druid:)
-          expect(service).to have_received(:delete).with(version: 2)
+        context 'when a version is provided' do
```

Comment:
Is this the "withdraw a version" case?  Can we change this context to a description "withdrawing a version".

Reply:
This will likely go away or get reworded as we don't have the "delete a version" use case.